internal/testdata/test-config.json

```diff
@@ -20,8 +20,9 @@
   ],
   "FieldPropagators": [
     {
-      "Receiver": "example.com/core.Source",
-      "AccessorRE": "^GetData$",
+      "PackageRE": "^example.com/core$",
+      "ReceiverRE": "Source",
+      "MethodRE": "^GetData$",
       "TODO": "Make this explicit configuration unnecessary."
```

Comment:
Does this TODO apply only to `FieldPropagators`? Would you like me to make it into an Issue for discussion?

If I understand correctly, we should be able to identify field propagators automatically, i.e. a field propagator is a method on a `Source` type that returns a tainted value (by which I mean a `Source`, a `Source`'s field or a value derived from them).

Reply:
(I guess we should also be able to identify TransformingPropagators automatically, e.g. in the current context any function that has a `Source` as a parameter, eventually a call that receives a `Source` value).

Reply:
I think the TODO was specifically referencing FieldPropagators, like simple getters.  Some of this comes from auto-generated code, like the structs from generated by protobuf.

I think if the getter returns a type identifiable as a source, it's moot, since the call will be identified as a source in the current, type-based identification.  But a field getter that, e.g., returns a secret string or `interface{}` would go unnoticed, and these are the things we would want to detect.

We should be able to identify these, but haven't gotten to it yet.  Thus this lingering TODO.

Reply:
SG. I'll convert it to an issue once I do my TODO cleanup thing.